client/src/components/History/ContentItem/DatasetCollection/DscMenu.vue

```diff
@@ -1,18 +1,53 @@
 <template>
-    <b-dropdown no-caret right variant="link" size="sm" boundary="window" toggle-class="p-1">
```

Comment:
Three open PRs from three authors with changes to make this a priority menu 😆. I think I've isolated the changes from Assunta in 8457c29f67e7bcf830bee353b2792a30b1c18376 if you can rebase on top of that? I meant to open a PR that was just this.

Reply:
I think the PriorityMenu was dropped ultimately in the collapsed view, so I guess this is all moot at this point.